lib/api/2.0/pollers.js

```diff
@@ -48,8 +49,16 @@ var pollersLibByIdGet = controller(function(req) {
  * @apiGroup pollers
  * @apiSuccess {json} pollers list of pollers or an empty object if there are none.
  */
-var pollersGet = controller(function(req) {
-    return pollers.getPollers(req.query);
+var pollersGet = controller(function(req, res) {
+    var options = {
+        skip: req.swagger.query.$skip,
+        limit: req.swagger.query.$top
+    };
+
+    return pollers.getPollers(req.query, options)
```

Comment:
@anhou req.swagger.query contains all "in query" swagger parameters that are explicitly defined in the swagger parameters object.  req.query contains query parameters that are not explicitly listed.  The `var query = req.swagger ? req.swagger.query : req.query` convention is generally only used in services that need to support both 1.1 and 2.0 APIs.  In this case, I want to extract $skip and $top from the swagger query because they are explicitly defined parameters and I want to pass req.query to `getPollers` to allow filtering of pollers using a user-defined query.